src/script/script_error.cpp

```diff
@@ -87,6 +87,14 @@ const char *ScriptErrorString(const ScriptError serror)
         return "Public key is neither compressed or uncompressed";
     case SCRIPT_ERR_NONCOMPRESSED_PUBKEY:
         return "Using non-compressed public key";
+    case SCRIPT_ERR_CLEANSTACK:
+        return "P2SH script evaluation of script does not result in a clean stack";
+    case SCRIPT_ERR_MUST_USE_FORKID:
+        return "Schnorr sigs must use forkid sighash if forkid flag set";
```

Comment:
doesn't this error apply also to ECSDA scheme? I mean maybe in the code we are only using for schnorr but wouldn't SIGHASH_FORKID applies to all BCH signature scheme?

Reply:
We will make this change in a different PR...